registration-processor/pre-processor/registration-processor-introducer-validator-stage/src/main/java/io/mosip/registration/processor/stages/introducervalidator/IntroducerValidator.java

```diff
@@ -231,47 +236,44 @@ private void validateIntroducerBiometric(String registrationId, InternalRegistra
 	 * @param list                  biometric data as BIR object
 	 * @param individualType        user type
 	 * @param registrationStatusDto
+	 * @throws Exception 
+	 * @throws  
 	 * @throws SAXException
 	 * @throws ParserConfigurationException
 	 * @throws NoSuchAlgorithmException
 	 * @throws InvalidKeySpecException
 	 * @throws IOException                  Signals that an I/O exception has
 	 *                                      occurred.
-	 * @throws BaseCheckedException
 	 * @throws BiometricException
 	 */
 
 	private void validateUserBiometric(String registrationId, String userId, List<BIR> list, String individualType,
-			InternalRegistrationStatusDto registrationStatusDto)
-			throws IOException, CertificateException, NoSuchAlgorithmException, BaseCheckedException {
+			InternalRegistrationStatusDto registrationStatusDto) throws BaseCheckedException {
+		regProcLogger.info("validateUserBiometric call started for registrationId {}", registrationId);
+		boolean status = false;
+		try {
+			status = bioUtil.authenticateBiometrics(userId, individualType, list);
+		} catch (Exception e) {
+			registrationStatusDto.setLatestTransactionStatusCode(
+					registrationExceptionMapperUtil.getStatusCode(RegistrationExceptionTypeCode.AUTH_ERROR));
+			registrationStatusDto.setStatusCode(RegistrationStatusCode.FAILED.toString());
```

Comment:
You have added AUTH_ERROR as exception type code. Instead if you throw the exception from here and handle in the process() method then exception will be handled properly. Make sure bioUtil.authenticateBiometrics() method throws proper exceptions which is handled in the process() method. I think no need to handle exception here. This is the design we have been following.